docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/artifactory/index.md

```diff
@@ -19,6 +19,34 @@ reconciliation by the Kargo controller, which will execute the discovery of
 new artifacts from all repositories to which that `Warehouse` subscribes.
 :::
 
+## Special note for self-hosted Artifactory users
+
+:::info
+If you are not using a self-hosted Artifactory instance, skip to
+[the configuring the receiver](#configuring-the-receiver) section.
+:::
+
+In order for a webhook initiated `Warehouse` refresh to successfully occur,
+it is required that you set a <Hlt>Custom Base URL</Hlt> for your instance.
```

Comment:
It may be worthwhile to elaborate on this (only a little) by explaining that when this setting hasn't been configured, critical information will be missing from the webhook payloads.